src/behaviors/focusZone.ts

```diff
@@ -484,6 +509,23 @@ export function focusZone(container: HTMLElement, settings?: FocusZoneSettings):
         updateFocusedElement(event.target)
       }
     })
+    container.addEventListener(
+      'mousemove',
+      ({target}) => {
```

Comment:
Should we debounce this?

Reply:
I think `mousemove` already has some built in (tho very minor) throttling built into the browser. In order to have the best user experience, I think we need to respond to every event.  With that said, there is a short circuit [here](https://github.com/primer/components/pull/1308/files#diff-7db8dd22df056db4e87f4cae691dcded5aaec5a8d8392a541cdbc33bad844e66R376-R382) that prevents these events from triggering changes all the way out to active descendant callbacks if the active descendant _has not changed_